src/pages/Assets/AssetCards/StakingVaults/StakingVaultRow.tsx

```diff
@@ -0,0 +1,77 @@
+import { Flex, HStack } from '@chakra-ui/layout'
+import { Button, Skeleton, SkeletonCircle } from '@chakra-ui/react'
+import { Tag } from '@chakra-ui/tag'
+import qs from 'qs'
+import { Link, useLocation } from 'react-router-dom'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { RawText, Text } from 'components/Text'
+import { SupportedYearnVault } from 'context/EarnManagerProvider/providers/yearn/constants/vaults'
+import { useFetchAsset } from 'hooks/useFetchAsset/useFetchAsset'
+
+export const StakingVaultRow = ({
+  type,
+  provider,
+  vaultAddress,
+  tokenAddress,
+  chain,
+  isLoaded
+}: SupportedYearnVault & { isLoaded: boolean }) => {
+  const location = useLocation()
+  const asset = useFetchAsset({ chain, tokenId: tokenAddress })
+
+  /* @TODO: Get this from somewhere */
+  const apr = '0.05'
```

Comment:
APR come from the yearn api when you get an asset by token or vault id

Reply:
Need to think about how we want to do this some more, since we need to pass the chaintype to each instance of the API.